### DISCUSSION TITLE
MySQL metrics not showing up on custom dashboard, but they work on the Netdata UI

### DISCUSSION THREAD

**Discussion by ricsi114** [netdata/netdata#18782] [Q&A] [answered]
I'm currently testing out the custom dashboard feature of netdata and I wanted to try it with some key metrics that I would like to specifically monitor.
The default metrics are working on the custom dashboard without any problems, but when it comes to mysql metrics, more specifically if I want to check replication with mysql.slave_status, I get the error: mysql.slave_status: chart not found on url.
Interestingly, it works perfectly fine in the UI, so this is a little bit confusing for me.

I checked the API endpoint to list all available charts, but I also could not see the sql metrics.

Maybe someone else already encountered this problem.
If so, any help would be appriciated.

**Comment by ilyam8** [comment]
Hi, @ricsi114. The error is

>  mysql.slave_status: chart not found on url.

The chart with this ID does not exist.

**Reply by ricsi114** [reply]
In the UI it is definitely named mysql.slave_status:
![image](https://github.com/user-attachments/assets/6aee6060-4a5a-4455-a442-be4f8b3fbcec)
But if it is really not the real name of it, where can I find it?

**Reply by ilyam8** [reply, marked as answer]
That is the charts' context, not id. This is id

<img width="567" alt="Screenshot 2024-10-16 at 10 34 00" src="https://github.com/user-attachments/assets/44ae23dc-cf1c-430e-8298-8995dbff4f65">

**Reply by ricsi114** [reply]
Ah, so this was the issue!
Now it seems to be working correctly!
Thank you very much :)
